sdk/python/setup.py

```diff
@@ -95,6 +95,10 @@
     "pyspark>=3.0.0",
 ]
 
+POSTGRES_REQUIRED = [
```

Comment:
another note: you should update `requirements/py3.7-ci-requirements.txt`, as well as the files for 3.8 and 3.9 - see https://github.com/feast-dev/feast/pull/2349/files